src/CommunityToolkit.Maui.UnitTests/Converters/IsListNotNullOrEmptyConverter_Tests.cs

```diff
@@ -16,21 +16,21 @@ public class ListIsNotNullOrEmptyConverter_Tests : BaseTest
 
 	[Theory]
 	[MemberData(nameof(Data))]
-	public void ListIsNotNullOrEmptyConverter(object value, bool expectedResult)
+	public void IsListNotNullOrEmptyConverter(object value, bool expectedResult)
 	{
-		var listIsNotNullOrEmptyConverter = new ListIsNotNullOrEmptyConverter();
+		var listIsNotNullOrEmptyConverter = new IsListNotNullOrEmptyConverter();
 
-		var result = (bool)listIsNotNullOrEmptyConverter.Convert(value, typeof(ListIsNotNullOrEmptyConverter), null, CultureInfo.CurrentCulture);
+		var result = (bool)listIsNotNullOrEmptyConverter.Convert(value, typeof(IsListNotNullOrEmptyConverter), null, CultureInfo.CurrentCulture);
 
-		Assert.Equal(result, expectedResult);
+		Assert.Equal(expectedResult, result);
 	}
 
 	[Theory]
 	[InlineData(0)]
 	public void InValidConverterValuesThrowArgumenException(object value)
```

Comment:
such a minor point but

```suggestion
	public void InvalidConverterValuesThrowArgumenException(object value)
```